sentinelsat/sentinel.py

```diff
@@ -91,15 +91,18 @@ def _api2dhus_url(api_url):
     def _req_dhus_stub(self):
         try:
             resp = self.session.get(
-                self.api_url + "api/stub/version", auth=self.session.auth, timeout=self.timeout
+                self._api2dhus_url(self.api_url) + "api/stub/version",
+                auth=self.session.auth,
+                timeout=self.timeout,
             )
             resp.raise_for_status()
         except requests.exceptions.HTTPError as err:
+            # is this part of the code useful?
             self.logger.error("HTTPError: ", err)
             self.logger.error("Are you trying to get the DHuS version of APIHub?")
             self.logger.error("Trying again after conversion to DHuS URL")
             resp = self.session.get(
-                self._dhus2api_url(self.api_url) + "api/stub/version",
+                self._api2dhus_url(self.api_url) + "api/stub/version",
```

Comment:
What is the motivation for changing the order of the name?

Reply:
There was an error in the code. The function that we try to call located on line 88 is called `_api2dhus_url`, not `_dhus2api_url`.